### PR TITLE
Update Default collector name and Clobber property.

### PR DESCRIPTION
**Pull request by ajaiswals**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by ajaiswals** [comment]
@swiatekm-sumo  @perk-sumo  with reference to - https://sumologic.slack.com/archives/C01KD5GHQ5C/p1685006783323539?thread_ts=1684963201.061259&cid=C01KD5GHQ5C, I wanted to propose this update in documentation for customers to understand the default and how/when to use Clobber.

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 1]
```diff
@@ -42,6 +42,25 @@ It is recommended to maintain the configuration of all reusable components in `c
...
 Our [**App Catalog**](/docs/get-started/apps-integrations) provides a mechanism to create these configuration files using a simple UI form input. Learn More.
 
+
+## Default Collector Name
+
+Each collector name must be unique. Collector by default uses hostname of the machine where collector is installed as the Collector's name. By default,if you are installing a collector that would have the same hostname as an existing collector, the system automatically appends a 13-digit unix timestamp to the collector name.
+
+## Forcing a Collector's Name with Clobber
+
+Use the clobber flag as true if you want to delete the existing collector if a new collector is installed on a machine with same hostname. This is usefull for scenrio where a VM is deleted and initiaed again with same hostname. To enable this property true, create a file sumo-ot-clobber.yaml in otelcol-sumo/conf.d , add below configuration and restart your collector.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:52]:
```suggestion
Use the clobber flag as true if you want to delete the existing collector if a new collector is installed on a machine with same hostname. This is useful for scenario where a VM is deleted and initiated again with the same hostname. To enable this property true, create a file `sumo-ot-clobber.yaml` in `otelcol-sumo/conf.d`, add below configuration and restart your collector.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 2]
```diff
@@ -42,6 +42,25 @@ It is recommended to maintain the configuration of all reusable components in `c
...
+Use the clobber flag as true if you want to delete the existing collector if a new collector is installed on a machine with same hostname. This is useful for scenario where a VM is deleted and initiated again with the same hostname. To enable this property true, create a file `sumo-ot-clobber.yaml` in `otelcol-sumo/conf.d`, add below configuration and restart your collector.
+
+```yaml
+extensions:
+  sumologic:
+    clobber: true
+```
+
+:::important
+Using the clobber flag as true deletes (clobbers) any existing collector with the same collector name/hostname, so make sure that is what you want to do. Clobber is effective only before the new collector has been registered (activated) with Sumo Logic.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:61]:
```suggestion
Setting the clobber flag to `true` deletes (clobbers) any existing collector with the same collector name/hostname, so make sure that is what you want to do. Clobber is effective only before the new collector has been registered (activated) with Sumo Logic.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 3]
```diff
@@ -42,6 +42,25 @@ It is recommended to maintain the configuration of all reusable components in `c
...
 Our [**App Catalog**](/docs/get-started/apps-integrations) provides a mechanism to create these configuration files using a simple UI form input. Learn More.
 
+
+## Default Collector Name
+
+Each collector name must be unique. Collector by default uses hostname of the machine where collector is installed as the Collector's name. By default,if you are installing a collector that would have the same hostname as an existing collector, the system automatically appends a 13-digit unix timestamp to the collector name.
+
+## Forcing a Collector's Name with Clobber
+
+Use the clobber flag as true if you want to delete the existing collector if a new collector is installed on a machine with same hostname. This is useful for scenario where a VM is deleted and initiated again with the same hostname. To enable this property true, create a file `sumo-ot-clobber.yaml` in `otelcol-sumo/conf.d`, add below configuration and restart your collector.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:52]:
```suggestion
Set the clobber flag to `true` if you want to delete the existing collector where a new collector is installed on a machine with same hostname. This is useful for scenarios where a VM is deleted and initiated again with the same hostname. To enable this property, create a file `sumo-ot-clobber.yaml` in `otelcol-sumo/conf.d`, add below configuration and restart your collector.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**